### PR TITLE
Improve macfuse installation

### PR DESCRIPTION
**Pull request by Ironicbay**
New window looks like this:
Capture d’écran 2021-07-05 à 11 47 12

The `<a href>` link should redirect to a detailed guide with pictures, steps etc.

### CODE REVIEW

**Review by Max-7 — APPROVED** [review]

File: parsec/core/gui/tr/parsec_en.po [hunk 1]
```diff
@@ -677,11 +677,22 @@ msgstr ""
...
+"FUSE could not be loaded. \n"
+"Please check your installation."
+
+msgid "TEXT_LOGIN_ERROR_MACFUSE_NOT_AVAILABLE"
+msgstr ""
+"To finalize your installation, macFUSE must be installed for certain key "
+"features to function.<br><br>On macOS versions 11.0 onwards, it's necessary "
+"to allow macFUSE in security settings after installation, requiring a system "
+"reboot afterwards.<br><br>A detailed installation guide is available "
+"<a href=\"https://www.google.com/search?q=todo\">here</a>."
```
Comment (Max-7) [inline parsec/core/gui/tr/parsec_en.po:689]: Fix the URL.

File: parsec/core/gui/tr/parsec_fr.po [hunk 2]
```diff
@@ -719,12 +719,23 @@ msgstr ""
...
+"Merci de vérifier votre installation."
+
+msgid "TEXT_LOGIN_ERROR_MACFUSE_NOT_AVAILABLE"
+msgstr ""
+"Pour finaliser votre installation, macFUSE doit être installé pour permettre "
+"l'utilisation de certaines fonctions de Parsec.<br><br>Sur les versions 11.x "
+"de macOS, il est nécessaire d'autoriser macFUSE dans les paramètres de "
+"sécurité de votre Mac après installation, requérant un redémarrage de votre "
+"ordinateur.<br><br>Un guide d'installation détaillé est disponible "
+"<a href=\"https://www.google.com/search?q=%C3%A0+faire\">ici</a>."
```
Comment (Max-7) [inline parsec/core/gui/tr/parsec_fr.po:732]: Fix the URL.

File: parsec/core/gui/custom_dialogs.py [hunk 3]
```diff
@@ -345,8 +345,34 @@ def _on_button_clicked(self, button):
...
-    w = InfoWidget(message, button_text)
+    w = InfoWidget(message, button_text=button_text)
     d = GreyedDialog(w, title=None, parent=parent, hide_close=True)
     w.dialog = d
     w.button_ok.setFocus()
     return d.open()
+
+
+class InfoLinkWidget(QWidget, Ui_InfoWidget):
+    def __init__(self, message, dialog=None, button_text=None, url=None):
```
Comment (Max-7) [inline parsec/core/gui/custom_dialogs.py:356]: Make the URL mandatory.

**Review by Max-7 — CHANGES_REQUESTED** [review]

**Comment by Ironicbay** [comment]
Current widget look: 
Capture d’écran 2021-07-06 à 09 53 56

**Review by touilleMan — APPROVED** [review]
👍

**Pull request merged**